static/js/dashboard.js

```diff
@@ -1094,8 +1094,14 @@ $(document).ready(function () {
         success: function (result) {
           var t1 = Date.now()
           var t2 = Date.now()
+          var chart = $("#proposed-chart").highcharts()
           if (result && result.length) {
             createProposedChart(result)
+          } else {
+            if (chart !== undefined) {
+              chart.showNoData("No proposals have been made from the selected validator(s)")
+              chart.series[0].setData([]) // Clear the series data
```

Comment:
This is not enough. The chart has more than one series. Here is how to trigger a bug:

1. With you branch, navigate to the dashboard and add only the following two validators: `4835` and `483555`
FYI: Validator [4835](https://prater.beaconcha.in/validator/4835) is old and contains lots of proposals, [483555](https://prater.beaconcha.in/validator/483555)  is very young and (currently) does not include any proposals. If you test this later, validator `483555` may already include proposals, in that case you have to find another one.
2. View the proposals tab, it might look similar to this:
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/114066135/7cf5c653-55ce-486d-8c28-fba617a82a98)
3. Remove validator `4835`. At this point, the chart should not show any data yet it looks like this:
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/114066135/4f93c950-64fd-415c-8e25-1fbc3daef348)

Reply:
I replaced the series for all 3 potential states of a proposal.